src/core/gsregisters.cpp

```diff
@@ -60,11 +60,11 @@ void GS_REGISTERS::write64_privileged(uint32_t addr, uint64_t value)
             printf("MAGV: %d\n", DISPLAY2.magnify_y);
             break;
         case 0x00E0:
-            printf("[GS_r] Write BGCOLOR: $%08X\n", value);
+            printf("[GS_r] Write BGCOLOR: $%08lX\n", value);
             BGCOLOR = value & 0xFFFFFF;
             break;
         case 0x1000:
-            printf("[GS_r] Write64 to GS_CSR: $%08X_%08X\n", value >> 32, value);
+            printf("[GS_r] Write64 to GS_CSR: $%08X_%08X\n", static_cast<uint32_t>(value >> 32), static_cast<uint32_t>(value));
```

Comment:
Instead of using `static_cast`, use the `$%08lX` format, as it's shorter and easier to read.